### PR TITLE
feat: updated TopicMessageQuery to have a default start time

### PR DESCRIPTION
**Pull request by andrix10**
Signed-off-by: Andrei Kuzmiankov <andrei@launchbadge.com>

closes: #414

### CODE REVIEW

File: topic_message_query.go [hunk 1]
```diff
@@ -27,11 +27,13 @@ type TopicMessageQuery struct {
 }
 
 func NewTopicMessageQuery() *TopicMessageQuery {
+	timeNow := time.Now()
 	return &TopicMessageQuery{
 		maxAttempts:       maxAttempts,
 		errorHandler:      _DefaultErrorHandler,
 		retryHandler:      _DefaultRetryHandler,
 		completionHandler: _DefaultCompletionHandler,
+		startTime:         &timeNow,
```
Comment (steven-sheehy) [inline topic_message_query.go:36]: This shouldn't be necessary. The server will default startTime to now if not sent. The problem must lie elsewhere. Do you have tests for this?

File: topic_message_query.go [hunk 2]
```diff
@@ -27,11 +27,13 @@ type TopicMessageQuery struct {
 }
```
Comment (steven-sheehy) [inline topic_message_query.go:27]: There's a bug in this file. It should be assigning body.ConsensusEndTime:
```go
	if query.endTime != nil {
		body.ConsensusStartTime = _TimeToProtobuf(*query.endTime)
	}
```

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: topic_message_query.go (hunk 2 above)
Comment (andrix10) [reply topic_message_query.go:27]: Thanks for catching this

File: topic_message_query.go (hunk 1 above)
Comment (andrix10) [reply topic_message_query.go:36]: Yed we had an e2e test, but it was actually erroring in the end with timeout. So far I am not sure at all what causes it, start time is nil whole time if not set. But if i just set it to empty it works.

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**